content/blog/pulumi-service-improvements_02-2020/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Pulumi Console Improvements, February 2020"
```

Comment:
```suggestion
title: "Administering Infrastructure Gets Easier with Pulumi Console Improvements"
```

Reply:
I'm not sure this improves, personally. I think the title works well as-is, given the informational nature of the post, along with the intent to release them on an ongoing basis. (In other words, in next month's batch, you probably wouldn't say exactly the same thing, right?) The suggestion also reads like a news headline or press release, which also doesn't quite line up with the content of the post. Just my 0.02, though. 😄

Reply:
If this is intended to be a progress update, then I agree with keeping the original title.